summer_staj.py

```diff
@@ -74,24 +87,26 @@ def parse(self, response):
 
         stuff = []
 
-        size = len(soup.find_all("td", class_="companyName"))

```

Comment:
constantly crawling a single web page with find_all is not great. Use it once keep what you have.